types/nodegit/reference.d.ts

```diff
@@ -5,9 +5,9 @@ import { Repository } from "./repository";
 export namespace Reference {
     const enum TYPE {
         INVALID = 0,
-        OID = 1,
+        DIRECT = 1,
         SYMBOLIC = 2,
-        LISTALL = 3,
+        ALL = 3,
```

Comment:
```suggestion
        /** @deprecated Use {@link ALL} instead. */
        LISTALL = 3,
        ALL = 3,
```

Reply:
thanks

Reply:
added in [df31899](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/72430/commits/df318993918ec769a05fad8c2cb821df4ae66078)